spec/features/friend_invitation_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 feature "friend invitation" do
-  scenario "user can invite a friend to join myflix" do
+  scenario "user can invite a friend to join myflix", js: true do
     jane = Fabricate :user
     sign_in_user jane
```

Comment:
@tomtomecek:
Now this call fails to sign `jane` in. I see a `incorrect username or password` flash message (i put a `sleep 10` in there to see that). That means line 5 is not creating `jane` in the test db.
This macro doesn't fail from other feature specs which are running headless. So the `js: true` is messing it up somehow.